recipes/inferno/LICENSE

```diff
@@ -0,0 +1,17 @@
+
+Apache Software License 2.0
```

Comment:
Actually it looks like this is already in `master`. So this file can be dropped (unless I'm missing something).

Reply:
Yes, the license file is included in the master already.
@marcelotrevisani asked me to include it in the recipe as well, so I did.
If it is not necessary, I can drop it again.